### ISSUE TITLE
Couldn't find path to game with bat file

### ISSUE THREAD

**Issue by ashesofdream** [soulsmods/ModEngine2#50]
I have put  all modengine file in directory where eldenring.exe exists.
powershell
``` 
..\ELDEN RING\Game>.\modengine2_launcher.exe -t er -c .\config_eldenring.toml
[2022-05-08 10:20:56.960] [stderr] [error] Couldn't find path to game
```

**Comment by Muhandes** [comment]
Same here

**Comment by namkazt** [comment]
make sure copy and paste modengine2 to game exe folder

edit `launchmod_eldenring.bat` content to this

> .\modengine2_launcher.exe -c .\config_eldenring.toml 

it will auto-detect exec in the current folder

**Comment by Eerye** [comment]
Same issue here. 

> edit `launchmod_eldenring.bat` content to this
> 
> > .\modengine2_launcher.exe -c .\config_eldenring.toml
> 
> it will auto-detect exec in the current folder

That's the exact string that's in the .bat by default. No matter what I change it to, even the exact location of the game returns an error because it still treats everything before the "\" like nothing is there or smth so it puts the path twice.
It also adds that ">" no matter what for whatever reason.

I think Windows 11 might have smth to do with this error. I have a different keyboard layout set, everything else is still English, yet for some reason certain parts of Windows still use English keyboard layout, such as the logon screen. And now I think the cmd might be affected in some way too even though I can type there in my layout. Or it might be due to Mi€ro$oft screwing around with the cmd in Win11, not sure. Buteverything worked fine in Win10.

If everyone with this error is on Win11 I guess we've found the source of the problem and just gotta wait for MS to fix their shit.

**Comment by namkazt** [comment]
im using windows 11.

the parameter `-t er` using for detect the games, if we not passing it, modengine will auto detect `eldenring.exe` in current folder of modengine2.exe and it should start

**Comment by Muhandes** [comment]
Using the above suggestion resolved the issue for me, thanks!
Note that you may need to run as Administrator

**Comment by Eerye** [comment]
> you not read it clear. there is no param `-t er`

Aah you're right, sorry, I'm an idiot. Thank you, works now!

**Comment by akihi21** [comment]
> Using the above suggestion resolved the issue for me, thanks! Note that you may need to run as Administrator
Hoe to solve this problem? I didn't get the point, by adding "t- er" in .bat file?
or to put modengine2.exe somewhere else?

**Comment by Muhandes** [comment]
>  Hoe to solve this problem? I didn't get the point, by adding "t- er" in .bat file?
>  or to put modengine2.exe somewhere else?

Extract all files to the game folder (e.g., C:\Games\Elden Ring\Game)

edit `launchmod_eldenring.bat` content to this

> .\modengine2_launcher.exe -c .\config_eldenring.toml

Run it as administrator. That did it for me and that's all I can do to help.

**Comment by awei3878** [comment]
> > 锄头解决这个问题？我不明白，通过在文件中添加“t-er”.bat？还是把modengine2.exe放在别的地方？
> 
> 将所有文件解压缩到游戏文件夹（例如，C：\Games\Elden Ring\Game）
> 
> 将内容编辑为此`launchmod_eldenring.bat`
> 
> > .\modengine2_launcher.exe -c .\config_eldenring.toml
> 
> 以管理员身份运行它。这为我做到了，这就是我所能做的一切。

对
`fea`

**Comment by awei3878** [comment]
> 我已将所有 modengine 文件放在 eldenring.exe 存在的目录中。动力壳
> 
> ```
> ..\ELDEN RING\Game>.\modengine2_launcher.exe -t er -c .\config_eldenring.toml
> [2022-05-08 10:20:56.960] [stderr] [error] Couldn't find path to game
> ```

qqq

**Comment by namkazt** [comment]
> > 我已将所有 modengine 文件放在 eldenring.exe 存在的目录中。动力壳
> > ```
> > ..\ELDEN RING\Game>.\modengine2_launcher.exe -t er -c .\config_eldenring.toml
> > [2022-05-08 10:20:56.960] [stderr] [error] Couldn't find path to game
> > ```
> 
> qqq

there is no `-t er` in parameters

**Comment by Viktor-2b** [comment]
using win11 and get this error,why?
Couldn't create process: 10b

**Comment by sorlax2410** [comment]
found a solution. 
..\ELDEN RING\Game>.\modengine2_launcher.exe -t er -c .\config_eldenring.toml
[2022-05-08 10:20:56.960] [stderr] [error] Couldn't find path to game
to adjust/change game path, you need to do is add --game-path <the path lead to your game> and don't leave space between elden ring for example if the name of your elden ring folder is ELDEN RING bring it to ELDENRING(this is due to programming rules). if you do want to leave space in between, you will need to find the special character that signifies it.

**Comment by namkazt** [comment]
hmm, i already explain `-t er` in bat file is use for auto-detect game.

if you not use it e.g: `..\ELDEN RING\Game>.\modengine2_launcher.exe -c .\config_eldenring.toml` it should work fine. maybe run with administrator.

**Comment by sorlax2410** [comment]
> hmm, i already explain `-t er` in bat file is use for auto-detect game.
> 
> if you not use it e.g: `..\ELDEN RING\Game>.\modengine2_launcher.exe -c .\config_eldenring.toml` it should work fine. maybe run with administrator.
> 
> this is the code part that explain what i was saying.
> 
> https://github.com/soulsmods/ModEngine2/blob/main/launcher/launcher.cpp
> 
> ![image](https://private-user-images.githubusercontent.com/6035916/344200443-c2fb9d94-9729-40e0-bbfa-fee13479bf1f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2MTEwMzIsIm5iZiI6MTcxOTYxMDczMiwicGF0aCI6Ii82MDM1OTE2LzM0NDIwMDQ0My1jMmZiOWQ5NC05NzI5LTQwZTAtYmJmYS1mZWUxMzQ3OWJmMWYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjhUMjEzODUyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YmI0ODkyYWIyYjkxZGQ2MTY2YmU1NDRhMzU2NDVkNjQ1NzJjN2Y3MjQ3YjdiNjdlMjZhOGVmNWJhOTE1ZWYyNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.i7pQGFtExxU7PEuePaBBbUYeVfmDbQcHqAsCwHJ2UtA) ![image](https://private-user-images.githubusercontent.com/6035916/344200537-df72463c-e71d-4c77-a018-bfd9844c0b70.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2MTEwMzIsIm5iZiI6MTcxOTYxMDczMiwicGF0aCI6Ii82MDM1OTE2LzM0NDIwMDUzNy1kZjcyNDYzYy1lNzFkLTRjNzctYTAxOC1iZmQ5ODQ0YzBiNzAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjhUMjEzODUyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZGYyMjJiNzUzYzExYjAxNjkyZjUyYzZkYmRkMGE5N2IxM2U2MWIxNTBkMWY2NDQyN2UzMDUzNDhkOTQxMTc5NSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.vmC2Jn_WjgdYGN4dR63K965_SnF0oB1tD3a5iEzMihc)
> 
> NOTE: exec name should be in this map ![image](https://private-user-images.githubusercontent.com/6035916/344201142-e9ff1e4b-78a2-43f0-ae9a-636e5df64fdf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2MTEwMzIsIm5iZiI6MTcxOTYxMDczMiwicGF0aCI6Ii82MDM1OTE2LzM0NDIwMTE0Mi1lOWZmMWU0Yi03OGEyLTQzZjAtYWU5YS02MzZlNWRmNjRmZGYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjhUMjEzODUyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NDNmZmViMDhkMTUzMTU2ZjIwNzRmZjNhOTNjMzI0NzUwYzBkYTI3NWRmOWYwMjM4OWRjY2Q3Yjc0MTQ2MzhlYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.e8A1bpl_M9ls-d9sgQuhRuI8DvIWg-VU2x9CZo_SLxw)

Yeah, about that, I tried everything you said, even with admin permission. It only works when I put it in the same game folder without the -t er parameters. When I put it outside, it doesn't work. It could be my PC so I did some lookup and found a solution on another page. Now I run into another problem, the game automatically create corrupted save files but only the shattered mod works. Everything behaves weirdly so I'm scraping by

**Comment by namkazt** [comment]
yeah it must put same game folder and without `-t er`, without it, it will detect from steam.

**Comment by sorlax2410** [comment]
Yah ... I kinda load it manually ... Now I'm wondering if that's why it causes the corrupted saves. I'll reinstall the game of needed....3rd time already
